config/config.yaml

```diff
@@ -26,6 +30,9 @@ resources:
   ontology:
     # gene ontology to download, used e.g. in goatools
     gene_ontology: "http://current.geneontology.org/ontology/go-basic.obo"
+  gencode:
+    gencodeBED: "https://sourceforge.net/projects/rseqc/files/BED/Human_Homo_sapiens/hg38_GENCODE.v38.bed.gz/download"
```

Comment:
What is this about?

Reply:
This was added for testing purposes and is now deleted.